docs/notebooks/10-sum-factorisation.ipynb

```diff
@@ -219,8 +219,10 @@
     "    fiat_make_rule = FIAT.quadrature.GaussLobattoLegendreQuadratureLineRule\n",
     "    fiat_rule = fiat_make_rule(FIAT.ufc_simplex(1), degree + 1)\n",
     "    finat_ps = finat.point_set.GaussLobattoLegendrePointSet\n",
-    "    finat_qr = finat.quadrature.make_quadrature\n",
-    "    return finat_qr(finat_ps(fiat_rule.get_points()), fiat_rule.get_weights())\n",
+    "    points = finat_ps(fiat_rule.get_points())\n",
+    "    weights = fiat_rule.get_weights()\n",
+    "    finat_qr = finat.quadrature.QuadratureRule\n",
+    "    return finat_qr(points, weights)\n",
```

Comment:
Are we adding anything by introducing the variable `finat_qr`? Maybe we should just use `finat.quadrature.QuadratureRule` directly.

Reply:
I guess not, will change now.